packages/utilities/psammead-locales/numerals.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./dist/numerals'); // eslint-disable-line import/no-unresolved
```

Comment:
For this to be ESM compatible this needs to be a package.json like https://github.com/bbc/psammead/blob/latest/packages/utilities/gel-foundations/scripts/package.json :)

Reply:
Thanks. Done. :)